classes/asserter/resolver.php

```diff
@@ -50,13 +55,28 @@ public function resolve($asserter)
 		{
 			$class = $this->checkClass($asserter);
 		}
-		else foreach ($this->namespaces as $namespace)
+		else
 		{
-			$class = $this->checkClass($namespace . '\\' . $asserter);
+			if (false === analyzer::isValidIdentifier($asserter))
```

Comment:
This is more complex actually. We can have a FQCN, like `my\asserters\foo`. So I will split the string by `\` and check each part if valids.